src/api/app/controllers/statistics/maintenance_statistics_controller.rb

```diff
@@ -3,7 +3,7 @@ class MaintenanceStatisticsController < ApplicationController
     skip_before_action :extract_user
 
     def index
-      @project = Project.find_by(name: params[:project])
+      @project = Project.find_by!(name: params[:project])
```

Comment:
I'm not sure this applies here but isn't that circumventing all the access checks? @adrianschroeter?